src/app/CommandHandler.cpp

```diff
@@ -196,10 +202,24 @@ CHIP_ERROR CommandHandler::ProcessCommandDataIB(CommandDataIB::Parser & aCommand
     err = commandPath.GetCommandId(&commandId);
     SuccessOrExit(err);
 
-    err = commandPath.GetEndpointId(&endpointId);
+    // TODO retrieve Endpoint ID with GroupDataProvider using GroupId and FabricId
+    // Issue 11075
+
+    // Using endpoint 1 for test purposes
```

Comment:
move comment down where the endpoint id is set? TODO should probably be there as well.

Reply:
done